src/Adapter/LocalStorage.php

```diff
@@ -197,6 +197,14 @@ public function getURL($path)
 
         return $this->webUrl.'/'.$path;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    public function getExpiringURL($path, $expiresAt) {
+        // Local storage cannot provide expiring URLs. Falling back...
```

Comment:
shoud't this Adaptor throw an exception to say 'feature not supported' or something similar in case someone call this method for localadaptor?

does not seam correct still allow this method to work and not do anything of what the method says it does.
might hide bugs or weird behaviour in your code.

If you're building this and use S3 adaptor making this throw an exception won't cause a problem right @zsturgess ?